release-note.html

```diff
@@ -23,22 +23,21 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
     <meta content="text/html; charset=iso-8859-1" http-equiv="content-type"/>
-    <title>Choreo Connect v1.0.0-Beta - 22th November 2021</title>
+    <title>Choreo Connect v1.0.0-RC1 - 6th December 2021</title>
```

Comment:
```suggestion
    <title>Choreo Connect v1.0.0 - 6th December 2021</title>
```

Reply:
@praminda this should be corrected right?

Reply:
yes, anything in the src should say 1.0.0. no RC1 should be there